lib/util/glob.js

```diff
@@ -24,40 +25,39 @@ const IGNORE = typeof Symbol === "function" ? Symbol("ignore") : "_shouldIgnore"
  * @param {function()} shouldIgnore Method to check whether a directory should be ignored.
  * @constructor
  */
-function GlobSync(pattern, options, shouldIgnore) {
-
-    /**
-     * We don't put this thing to argument `options` to avoid
-     * further problems, such as `options` validation.
-     *
-     * Use `Symbol` as much as possible to avoid confliction.
-     */
-    this[IGNORE] = shouldIgnore;
-
-    Sync.call(this, pattern, options);
-}
-
-util.inherits(GlobSync, Sync);
+class GlobSync {
+    constructor(pattern, options, shouldIgnore) {
+
+        /**
+         * We don't put this thing to argument `options` to avoid
+        * further problems, such as `options` validation.
+        *
+        * Use `Symbol` as much as possible to avoid confliction.
+        */
+        this[IGNORE] = shouldIgnore;
+        Sync.call(this, pattern, options);
```

Comment:
This line can be replaced with `super(pattern, options)` if `extends Sync` is used in the class declaration. (It would also have to be placed before the `this[IGNORE] = shouldIgnore;` line.)